content/en/agent/faq/circleci-incident-impact-on-datadog-agent.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Impact of the CircleCI Security Incident on the Datadog Agent
+kind: faq
+---
+
+On January 4th, 2023, we were notified by CircleCI that they were investigating a [security incident](https://circleci.com/blog/january-4-2023-security-alert/) that may have led to leaking stored secrets. We identified a single secret stored in CircleCI that could theoretically be misused by a potential attacker, an old RPM GNU Privacy Guard (GPG) private signing key and its passphrase. This page documents the implications of the potential leak and the measures we are taking to mitigate any risks to our customers.
+
+<div class="alert alert-info">
+<strong>Note</strong>: As of January 10th, 2023, we have no indication that the key was actually leaked or misused, but we are still taking the following actions out of an abundance of caution.
+</div>
+
+## The Affected Key
+
+The RPM GPG signing key has fingerprint `4172A230`, and is accessible in [our signing keys location](https://keys.datadoghq.com/DATADOG_RPM_KEY.public). This key was historically used to sign Agent 5 releases and Agent 6 releases up to (and including) 6.13.0.
```

Comment:
Something I just thought about (can be addressed later): should we use the full fingerprint of the key, to be in line with what we do in other docs pages (like https://docs.datadoghq.com/data_security/agent/)?
```suggestion
The RPM GPG signing key has fingerprint `60A389A44A0C32BAE3C03F0B069B56F54172A230`, and is accessible in [our signing keys location](https://keys.datadoghq.com/DATADOG_RPM_KEY.public). This key was historically used to sign Agent 5 releases and Agent 6 releases up to (and including) 6.13.0.
```